guide/src/sextant/npk_format_details.md

```diff
@@ -0,0 +1,239 @@
+# The NPK File Format
+
+NPKs are compressed archives containing both the container's application logic as well as the data files necessary to mount an run the container.
+
+Internally, an NPK consist of three files:
+
+- manifest.yaml
+- signature.yaml
+- fs.img
+
+## Manifest.yaml
+
+The file `manifest.yaml` references all data necessary to mount and execute the container.
+As an example, the `manifest.yaml` of the `memeater` example container looks as follows:
+
+```yaml
+name: memeater
+version: 0.0.1
+init: /memeater
+uid: 1000
+gid: 1000
+mounts:
+    /lib:
+      host: /lib
+    /lib64:
+      host: /lib64
+    /system:
+      host: /system
+cgroups:
+  memory:
+    limit_in_bytes: 10000000
+    swappiness: 0
+io:
+  stdout:
+    log:
+      - DEBUG
+      - memeater
+```
```

Comment:
probably better for that one since the first manifest does not even need to be completely visible in the first place